pims/image_sequence.py

```diff
@@ -94,6 +101,20 @@ def __init__(self, path_spec, process_func=None, dtype=None,
         else:
             self._dtype = dtype
 
+    def close(self):
+        if self._is_zipfile:
+            self._zipfile.close()
+
+    def __del__(self):
```

Comment:
I am not sure if it's a good idea to define the destructor to call `close` in all cases. If you call `frames.close()`, and then destroy the object, `close()` will be run twice which might give trouble for some readers.

For the zipfile it's no problem to close it twice, that's why I put it in for this specific reader.

`__enter__` `__exit__` already works by defining `close()`, right?